spec/ripple/callbacks_spec.rb

```diff
@@ -46,6 +46,15 @@
       callbacks.should == [ :before, :around, :after ]
     end
 
+    it 'halts the callback chain when false is returned' do
```

Comment:
The other tests in this file use bunny quotes for the `it` calls.

Reply:
It's pretty mixed from what I'm looking at - going to continue with the mixed quotes.